pages/common/typeinc.md

```diff
@@ -0,0 +1,17 @@
+# typeinc
+
+> Typeinc, an ncurses based terminal typing speed test program, written in python.
+> Try out different difficulty levels and improve your typing speed.
+> More information: <https://github.com/AnirudhG07/Typeinc>.
+
```

Comment:
Can you please add an example for entering the typing test?
```suggestion

- Enter the typing test:
`typeinc`

```

Reply:
Sure